.eslintrc.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
     '@typescript-eslint/no-namespace': 'off',
     '@typescript-eslint/explicit-function-return-type': 'off',
     '@typescript-eslint/no-explicit-any': 'off',
+    '@typescript-eslint/no-use-before-define': 'off',
```

Comment:
Thank god!! We are getting rid of this rule 😅. Can we remove all the comments related to this rule?

Reply:
yeah - i'll put another PR up to do that